lib/nagware.ts

```diff
@@ -1,62 +1,83 @@
 import { NagwareSubmission, NagwareResult, NagLevel } from "@lib/types";
+import { getAppSetting } from "./appSettings";
+import { logMessage } from "./logger";
 
-export function detectOldUnprocessedSubmissions(submissions: NagwareSubmission[]): NagwareResult {
-  const currentDate = Date.now();
+export async function detectOldUnprocessedSubmissions(
+  submissions: NagwareSubmission[]
+): Promise<NagwareResult> {
+  try {
+    const currentDate = Date.now();
+    const [promptPhaseDays, warnPhaseDays] = await Promise.all([
+      getAppSetting("nagwarePhasePrompted"),
```

Comment:
Should probably default to something if this fails?

Reply:
There is a check just below so that if either promptPhaseDays or warnPhaseDays are null/undefined the function immediately returns a `NagLevel.None` so the client will not show any messages.  It's like if the feature is turned off.